server/manager/src/main/java/org/apache/accumulo/manager/upgrade/UpgradeCoordinator.java

```diff
@@ -152,8 +155,13 @@ public synchronized void upgradeZookeeper(ServerContext context,
         abortIfFateTransactions(context);
 
         for (int v = currentVersion; v < AccumuloDataVersion.get(); v++) {
-          log.info("Upgrading Zookeeper from data version {}", v);
-          upgraders.get(v).upgradeZookeeper(context);
+          log.info("Upgrading Zookeeper from data version {} target version {}", v,
+              AccumuloDataVersion.get());
+          var upgrader = upgraders.get(v);
+          log.info("UPGRADER: version:{} upgrader: {}", v, upgrader);
```

Comment:
Is this expecting the upgrader to have a meaningful toString? If we want it to log here, in a consistent way, we should add a dedicated method for getting the description for logging. Or, we can create an upgrader base class that logs itself when run.

Reply:
The was leftover from development - addressed with b3ca1d9b65